src/devices/tuya.ts

```diff
@@ -3571,13 +3570,71 @@ const definitions: Definition[] = [
                 [4, 'current_heating_setpoint', tuya.valueConverter.divideBy10],
                 [5, 'local_temperature', tuya.valueConverter.divideBy10],
                 [7, 'child_lock', tuya.valueConverter.lockUnlock],
-                [28, 'schedule_monday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(1)],
-                [29, 'schedule_tuesday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(2)],
-                [30, 'schedule_wednesday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(3)],
-                [31, 'schedule_thursday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(4)],
-                [32, 'schedule_friday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(5)],
-                [33, 'schedule_saturday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(6)],
-                [34, 'schedule_sunday', tuya.valueConverter.thermostatScheduleDayMultiDPWithDayNumber(7)],
+                [28, 'schedule_monday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 1)],
+                [29, 'schedule_tuesday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 2)],
+                [30, 'schedule_wednesday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 3)],
+                [31, 'schedule_thursday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 4)],
+                [32, 'schedule_friday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 5)],
+                [33, 'schedule_saturday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 6)],
+                [34, 'schedule_sunday', tuya.valueConverter.thermostatScheduleDayMultiDPNWithDayNumber(4, 7)],
+                [35, null, tuya.valueConverter.errorOrBatteryLow],
+                [36, 'frost_protection', tuya.valueConverter.onOff],
+                [39, 'scale_protection', tuya.valueConverter.onOff],
+                [47, 'local_temperature_calibration', tuya.valueConverter.localTempCalibration2],
+            ],
+        },
+    },
+    {
+        fingerprint: tuya.fingerprint('TS0601', [
+            // '_TZE200_bvu2wnxz', /* model: 'ME167', vendor: 'AVATTO' */
+            // '_TZE200_6rdj8dzm', /* model: 'ME167', vendor: 'AVATTO' */
+            '_TZE200_p3dbf6qs', /* model: 'ME167', vendor: 'AVATTO' */
+            // '_TZE200_rxntag7i', /* model: 'ME168', vendor: 'AVATTO' */
```

Comment:
The patch worked for me with the fingerprint _TZE200_p3dbf6qs, I have no idea for the other devices. I will ask around in the ticket if someone needs an additional fingerprint here